routes/index.js

```diff
@@ -181,4 +181,54 @@ router.post('/register', function(req, res, next) {
   }
 });
 
+
+/* POST register-admin - try to register a new admin */
+router.post('/register-admin', (req, res) => {
+  // check the request is originating from localhost...
+  if (req.headers.host.split(':')[0] === 'localhost') {
+
+    // check if all fields are filled
+    if (req.body.username && req.body.password && req.body.password_confirm) {
```

Comment:
You don't have to rebuild this - check out the `registerAdmin` function in `auth.js`

Reply:
nvm ignore this LOL.